aria/cli/helptexts.py

```diff
@@ -47,3 +47,8 @@
 SORT_BY = "Key for sorting the list"
 DESCENDING = "Sort list in descending order [default: False]"
 JSON_OUTPUT = "Output logs in a consumable JSON format"
+
+DISPLAY_JSON = "Display in JSON format"
```

Comment:
lets merge this with `JSON_OUTPUT`,
the latter is a good name, and simply have it as "Format the output as JSON"